temporalnexus/operation.go

```diff
@@ -332,17 +340,11 @@ func ExecuteUntypedWorkflow[R any](
 						Header: nexusOptions.CallbackHeader,
 					},
 				},
+				Links: links,
 			},
 		})
 	}
 
-	links, err := convertNexusLinks(nexusOptions.Links, GetLogger(ctx))
-	if err != nil {
-		return nil, &nexus.HandlerError{
-			Type:  nexus.HandlerErrorTypeBadRequest,
-			Cause: err,
-		}
-	}
 	internal.SetLinksOnStartWorkflowOptions(&startWorkflowOptions, links)
```

Comment:
Please add a note saying that the links are sent twice for older servers that don't accept links in the callback.